src/hardware/pcspeaker.cpp

```diff
@@ -81,36 +82,23 @@ static struct {
 
 	bool pit_mode3_counting = false;
 	float volwant, volcur = 0.0f;
-	// Bitu last_ticks;
 	float last_index = 0.0f;
 	int minimum_counter = 0;
-	DelayEntry entries[SPKR_ENTRIES] = {};
-	int used = 0;
+	uint16_t tally_of_silence = 0;
 } spkr = {};
 
-inline static void AddDelayEntry(const float index, const int16_t new_output_level)
+static void add_impulse(const DelayEntry &entry);
```

Comment:
Just a style question: I've noticed some functions are `lower_underscore_case` while some are `UpperCamelCase`? When would you pick one over the other?

Reply:
This is another style inconsistency especially in this PC speaker code. I'll fix this when refactoring it all into a class. 
In general, 

DOSBox exports public functions as:
DEVICE_SomeFeature(...)

Then static non-class functions are lower_snake(), while class functions are CameCase().

We try to follow the most consistent styles found in DOSBox, but some areas have  inconsistencies (like this one). 

GUS is a good example of a fully refactored device (sample with Tandy).

Reply:
Also checkout https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#coding-style

(In general, Staging tries to codify and standardize around the most common upstream styles found in DOSBox, so existing authors feel at home and to make cross-contriburing to SVN and X as easy as possible).

Reply:
Cheers, I read that page last year, completely forgot about it. Time to revisit!